static/mixins/polling.jsx

```diff
@@ -6,11 +6,67 @@ var PollingMixin = {
       pollingActive: true
     };
   },
+  pushNotification(){
+    let body;
+    let {app, env, number} = this.props.params;
+    const url = `/deploys/${app}/${env}/${number}`;
 
-  componentWillMount() {
+    if(this.state.task !== undefined){
+      body = `${/[^@]*/.exec(this.state.task.user.name)}'s deploy ${this.state.task.status}`;
+    }else{
+      body = `${/[^@]*/.exec(this.state.deploys[0].user.name)}'s deploy ${this.state.deploys[0].status}`;
+    }
+
+    const options = {
+      body: body,
+      icon: "../static/favicon.png"
```

Comment:
I wouldn't make this relative with `../static/`. Just use `/static/`. This will break if it were ever on a url that wasn't this exact structure.